cinema/models.py

```diff
@@ -35,12 +40,20 @@
         return f"{self.first_name} {self.last_name}"
 
 
+def uploads_movies(instance, filename):
+    _, extension = os.path.splitext(filename)
+    filename = f"{slugify(instance.title)}--{uuid.uuid4()}.{extension}"
+
+    return os.path.join("uploads", "images", filename)
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to="uploads/images/")
```

Comment:
The `image` field in the `Movie` model should use the `uploads_movies` function for the `upload_to` parameter to ensure that images are stored with the correct filename format. Change `upload_to="uploads/images/"` to `upload_to=uploads_movies`.